.github/workflows/fsanitize-check.yml

```diff
@@ -7,39 +7,73 @@ on:
     branches: [ '*' ]
 
 jobs:
-  build:
+  build_wolfssl:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ ubuntu-latest ]
+        config: [
+          # Add new configs here and make wolfclu matrix match
+          '--enable-wolfclu',
+          '--enable-wolfclu --enable-smallstack',
+          '--enable-wolfclu --enable-experimental --enable-dilithium',
+          '--enable-wolfclu --enable-smallstack --enable-experimental --enable-dilithium',
+        ]
+    name: Build wolfssl
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 4
+    steps:
+      - name: Checking cache for wolfssl
+        uses: actions/cache@v4
+        id: cache-wolfssl
+        with:
+          path: build-dir/
+          key: wolfclu-fsanitize-check-wolfssl-${{ strategy.job-index }}-${{ matrix.os }}
+          lookup-only: true
 
-    runs-on: ubuntu-latest
+      - name: Checkout, build, and install wolfssl
+        if: steps.cache-wolfssl.outputs.cache-hit != 'true'
+        uses: wolfSSL/actions-build-autotools-project@v1
+        with:
+          repository: wolfssl/wolfssl
+          ref: master
+          path: wolfssl
+          configure: ${{ matrix.config }} CC="gcc -fsanitize=address"
+          check: false
+          install: true
 
+  build_wolfclu:
+    needs: build_wolfssl
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ ubuntu-latest ]
+        config: [
+          '--enable-wolfclu',
+          '--enable-wolfclu --enable-smallstack',
+          '--enable-wolfclu --enable-experimental --enable-dilithium',
+          '--enable-wolfclu --enable-smallstack --enable-experimental --enable-dilithium',
+        ]
+    name: Build wolfclu
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 4
     steps:
-    - uses: actions/checkout@master
-      with:
-        repository: wolfssl/wolfssl
-        path: wolfssl
-    - name: wolfssl autogen
-      working-directory: ./wolfssl        
-      run: ./autogen.sh
-    - name: wolfssl configure
-      working-directory: ./wolfssl 
-      run: ./configure --enable-wolfclu --enable-crl --enable-dsa --enable-pkcs7
```

Comment:
Are we losing coverage for CRL & DSA?

Reply:
good point, will add in a build for CRL & DSA